bot/cogs/help.py

```diff
@@ -110,9 +110,9 @@ def _get_query(self, query: str) -> Union[Command, Cog]:
         cog = self._bot.cogs.get(query)
         if cog:
             return Cog(
-                name=cog.__class__.__name__,
+                name=cog.cog_name,
```

Comment:
`cog_name` the wrong attribute for Cogs (but not for Commands) and will raise an error
```suggestion
                name=cog.qualified_name,
```

Reply:
Under what circumstance does this raise an exception?

Reply:
It's an AttributeError. So almost always whenever a cog name is given to the help command (in my testing `Moderation` still worked for some reason).

Reply:
Weird, worked fine when I was looking:

![image](https://user-images.githubusercontent.com/5323929/65789676-de3e9f00-e12b-11e9-9e9b-c03158378e80.png)